docs/api/en/math/Box3.html

```diff
@@ -129,7 +129,7 @@ <h3>[method:Box3 expandByObject]( [param:Object3D object] )</h3>
 		Expands the boundaries of this box to include [page:Object3D object] and its children,
 		accounting for the object's, and children's, world transforms.
 		The function may result in a larger box than strictly necessary.
-
+		It's possible to get the precise box by using [page:.setFromBufferAttribute]() if desired.
```

Comment:
How do you account for the matrix transforms?

Reply:
>The function may result in a larger box than strictly necessary.

Instead, I would say:

    Note: For computational efficiency, the computed bounding box is conservative. That is, it is not necessarily the minimal bounding box.

Reply:
I'm afraid this note does not recommend what to do when you the exact bounding box. The was the actual problem of the OP in #19030.

> How do you account for the matrix transforms?

Well, you could clone the position attribute and then use `BufferAttribute.applyMatrix4()`.

Reply:
You are not using `Box3.setFromBufferAttribute()` as you proposed.

I would just omit the comment you are attempting to add here. Someone decided we are not going to return the minimal bounding box, so forget it.

Reply:
Well, it was the feature request of the OP. I let @mrdoob decide if the comment should be added. The new comment points user into the right direction.

Reply:
I am trying to help you.

As far as I can see, your comment is false.

Reply:
> As far as I can see, your comment is false.

Can you explain why?

Reply:
>It's possible to get the precise box by using [page:.setFromBufferAttribute]()

It requires doing something this:

```js
var attribute = attributes.position.clone();

attribute.applyMatrix4( object.matrixWorld );

box3.setFromBufferAttribute( attribute );
```
Getting it to work on a hierarchy of objects will require writing another method.

So the statement in the docs is not helpful. In fact, it is misleading. I would simply remove it.

Reply:
Don't you think users can figure out by themselves if it's necessary to honor the world matrix or the object hierarchy?  Mentioning that a solution needs use `setFromBufferAttribute()` or `expandByPoint()` is better than nothing from my point of view.

Reply:
The statement is misleading and certainly not sufficiently informative to be useful.

Look how much trouble I had trying to figure out what would be required -- and we have not even specified the entire program yet.

I would remove the statement.

Reply:
If you think it's not useful then I close the PR. We can still add something to the docs or the FAQ if more devs run into this issue. Feel free to change the existing text if you like. I think both are fine.

Reply:
@Mugen87 Thank you for considering my suggestion. :-)

I agree with you that we can leave it as-is for now.